staging/src/k8s.io/apimachinery/pkg/api/meta/restmapper.go

```diff
@@ -519,3 +519,12 @@ func (m *DefaultRESTMapper) RESTMappings(gk schema.GroupKind, versions ...string
 	}
 	return mappings, nil
 }
+
+// MaybeResetRESTMapper calls Reset() on the mapper if it is a ResettableRESTMapper.
+func MaybeResetRESTMapper(mapper RESTMapper) bool {
```

Comment:
naive question, it seems the `bool` output is never used , is it worth having it?

Reply:
It's not used at the moment, but may be used in the future, including by users of this library. I'm more than happy to remove it if this will increase chances of the PR getting merged.

Reply:
no no, I was curious only, wait for the maintainers opinion

Reply:
let me ping @liggitt

Reply:
My preference is to not have it.

Reply:
> My preference is to not have it.

Given timing, I'm willing to take the removal of this bool as a followup.

Reply:
https://github.com/kubernetes/kubernetes/pull/106472